src/utils/hotspots/generate-hotspots-configs.js

```diff
@@ -15,12 +15,14 @@ export const generateHotspotsConfigs = (hotspotsProps) => {
     configsErrorHandler(hotspotProps);
 
     const popupConfig = generatePopupConfig(popupProps);
+    const uniqueID = Math.floor(Math.random() * 10000);
```

Comment:
as uniqueID will be used just if variant.anchorId doesn't exist so better to wrap it inside that condition too.